app/actions/views/channel.js

```diff
@@ -552,30 +558,34 @@ export function setChannelDisplayName(displayName) {
 }
 
 // Returns true if there are more posts to load
-export function increasePostVisibility(channelId, focusedPostId) {
+export function increasePostVisibility(channelId, postId) {
     return async (dispatch, getState) => {
         const state = getState();
-        const {loadingPosts, postVisibility} = state.views.channel;
+        const {loadingPosts} = state.views.channel;
```

Comment:
not that it matters much, but wanted to make sure why changing this?

Reply:
Oh, I should revert that. The `post-list-improvements` branch had the section using `postVisibility` commented out because of the bi-directional scrolling changes, and I forgot to recombine the lines when it was uncommented